scripts/modifier.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+PKG = 'embodied_attention'
```

Comment:
modifier.py looks the same as curiosity.py
if you want to make a generic mechanism to modify the saliency map, it should inherit from it, but this is not necessary.
What is modifier.py doing?